index.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
 module.exports = {
-  name: require('./package').name
+  name: require('./package').name,
+
+  included(app) {
+    this._super.included(...arguments);
```

Comment:
I think this needs to be `this._super.included.call(this, app);` -- as-is, when I run tests I get `TypeError: WeakMap is not a constructor` from [here](https://github.com/kellyselden/ember-macro-helpers/blob/master/addon/create-class-computed.js#L25) because `Ember.WeakMap` is `undefined` -- I assume because `ember-weakmap`'s `included` isn't being invoked properly by this call.

Reply:
I fixed the included, call. Good catch on the weakmap, I didn't intent to regress like that. I guess since I already accidentally regressed [here](https://github.com/kellyselden/ember-macro-helpers/pull/260/files#diff-b773593af4b01d438ca5bdef65ebf731R1) and WeakMap works in IE11, I suppose we can just leave it?